nonuke/nonuke.py

```diff
@@ -157,18 +157,33 @@ async def action(self, ctx, action: str):
         await self.initialize(ctx.guild)
 
     @nonuke.command()
-    async def whitelist(self, ctx, user: discord.Member):
-        """Add/Remove users from the whitelist"""
-        async with self.config.guild(ctx.guild).whitelist() as whitelist:
-            if user.id in whitelist:
-                whitelist.remove(user.id)
-                await ctx.send(f"{user} has been removed from the whitelist!")
-            else:
-                whitelist.append(user.id)
-                await ctx.send(f"{user} has been added to the whitelist!")
+    async def whitelist(
+        self, ctx: commands.Context, add_or_remove: commands.Literal["add", "remove"], users: commands.Greedy[discord.Member] = None
+    ):
+        """
+        Add/Remove users from the whitelist
+        
+        `<add_or_remove>` should be either `add` to add users or `remove` to remove users.
+        """
+        if users is None:
+            return await ctx.send("`Users` is a required argument.")
+        
+        async with self.config.guild(ctx.guild).whitelist() as wl:
+            for user in users:
+                if add_or_remove.lower() == "add":
+                    if not user.id in wl:
+                        wl.append(user.id)
+                        
+                elif add_or_remove.lower() == "remove":
+                    if user.id in wl:
+                        wl.remove(user.id)
+                        
+        return await ctx.send(
+            f"Successfully {'added' if add_or_remove.lower() == 'add' else 'removed'} {len([user for user in users])} users {'to' if add_or_remove.lower() == 'add' else 'from'} the whitelist."
```

Comment:
Please make this handle plural and singular, for example if only adding 1 user it would not be `users`